core/domain/skill_domain.py

```diff
@@ -787,14 +787,17 @@ def update_explanation(self, explanation):
         """Updates the explanation of the skill.
 
         Args:
-            explanation: dict. The new explanation of the skill.
+            explanation: SubtitledHtml.
+            The representation of the explanation of the skill.
```

Comment:
Move this to the line above.

Reply:
Hi Sandeep , when i merged the two lines 790 and 791  i got linker error for number of character in a line

Reply:
skill_services .py has taken care of the corresponding backend test line 480 in skill_services.py

Reply:
Done